addons/project/static/src/js/project_list.js

```diff
@@ -39,14 +39,44 @@ const ProjectListController = ListController.extend({
         return this._super(...arguments);
     },
 
-    _stopRecurrence(recurringResIds, resIds, mode) {
+    _countRecords(recurrenceIds, resIds) {
+        return this._rpc({
+            model: 'project.task',
+            method: 'read_group',
+            args: [
+                [['recurrence_id', 'in', recurrenceIds], ['id', 'not in', resIds]],
+                ['ids:count(id)'],
+                ['recurrence_id']
+            ],
+        });
+    },
+
+    async _stopRecurrence(recurringResIds, resIds, mode) {
+        const recurrenceIds = [];
+        for (const record of this.getSelectedRecords()) {
+            const recurrenceId = record.data.recurrence_id;
+            if (recurrenceId && !recurrenceIds.includes(recurrenceId)) {
+                recurrenceIds.push(recurrenceId);
+            }
+        }
```

Comment:
We could perhaps use a Set to be sure we don't add duplicate recurrence_id and thus we can avoid checking this: `!recurrenceIds.includes(recurrenceId)`. ^^
```suggestion
        const recurrenceIdsSet = new Set();
        for (const record of this.getSelectedRecords()) {
            const recurrenceId = record.data.recurrence_id;
            if (recurrenceId) {
                recurrenceIdsSet.add(recurrenceId);
            }
        }
        recurrenceIds = Array.from(recurrenceIdsSet);
```